src/aid.rs

```diff
@@ -56,16 +56,16 @@ impl core::fmt::Debug for Aid {
     fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
         if self.len <= self.truncated_len {
             f.write_fmt(format_args!(
-                "'{} {}'",
-                hexstr!(&self.bytes[..5]),
-                hexstr!(&self.bytes[5..self.len as _])
+                "'{:X?} {:X?}'",
+                &self.bytes[..5],
+                &self.bytes[5..self.len as _]
             ))
         } else {
             f.write_fmt(format_args!(
-                "'{} {} {}'",
-                hexstr!(&self.bytes[..5]),
-                hexstr!(&self.bytes[5..self.truncated_len as _]),
-                hexstr!(&self.bytes[self.truncated_len as _..self.len as _])
+                "'{:X?} {:X?} {:X?}'",
+                &self.bytes[..5],
+                &self.bytes[5..self.truncated_len as _],
+                &self.bytes[self.truncated_len as _..self.len as _]
```

Comment:
Actually if we remove `hexstr!` here we can fully remove delog.

I don't want to have the aid show up as an array. Please just use a loop.


```rust
            f.write_str("'")?;
            for b in &self.bytes[..5] {
                f.write_fmt(format_args!("{:02X}", b))?;
            }
            f.write_str(" ")?;
            for b in &self.bytes[5..self.len as _] {
                f.write_fmt(format_args!("{:02X}", b))?;
            }
            f.write_str("'")?;
            Ok(())
```